share/settings/ldap.go

```diff
@@ -0,0 +1,133 @@
+package settings
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/go-ldap/ldap/v3"
+)
+
+//LdapConfig enables LDAP auth
+type LdapConfig struct {
```

Comment:
can we replace all capitalised acronym identifiers `LdapFooBar` with `LDAPFoobar` ?

Reply:
changed as requested